src/components/SearchResults/TableHead.jsx

```diff
@@ -1,14 +1,38 @@
 import React from "react";
 
-const TableHead = () => {
+const TableHead = ({ handleSort, sortConfig }) => {
   return (
     <tr>
-      <th>ID</th>
-      <th>Title</th>
-      <th>First Name</th>
-      <th>Last Name</th>
-      <th>Nights</th>
-      <th>Email</th>
+      <th onClick={() => handleSort("id")}>
+        ID{" "}
+        {sortConfig.key === "id" &&
+          (sortConfig.direction === "ascending" ? "▲" : "▼")}
```

Comment:
Very creative. loved it!!! you can set "id ▼" as default as it is how to list sorted when default.